tests/integration/test_pep517.py

```diff
@@ -75,6 +77,11 @@ def test_pip_wheel_build(temporary_directory: Path, project_source_root: Path) -
     assert len(wheels) == 1
 
 
+@pytest.mark.xfail(
+    tuple(int(part) for part in platform.python_version_tuple()[:2]) < (3, 8),
```

Comment:
What about using `sys.version_info < (3, 8)` like in [pytest documentation](https://docs.pytest.org/en/stable/how-to/skipping.html#id1)?

Reply:
I knew there was a simpler way to do that but couldn't remember. 👍